features/grpc/exporter/pom.xml

```diff
@@ -161,5 +160,155 @@
             <artifactId>protobuf-java</artifactId>
             <version>${protobufVersion}</version>
         </dependency>
+        <dependency>
```

Comment:
Always try to see that test dependencies are at the bottom with scope test, Don't mix and match